Dockerfile

```diff
@@ -8,7 +8,7 @@ RUN pip install poetry \
     && poetry config virtualenvs.create false
 RUN poetry install --no-root --no-dev
 COPY . /src/
-ENTRYPOINT ["gunicorn", "--bind", "0.0.0.0", "wsgi:app"]
+ENTRYPOINT "gunicorn --bind 0.0.0.0:$PORT wsgi:app"
```

Comment:
Your docker-compose.prod.yml is now out of date because this _requires_ a $PORT env var.

You should either set a default here, or update that file. 

And `EXPOSE 5000` should be moved from the "base" to the "development" stage.